src/TableRecordInstantiator.php

```diff
@@ -38,12 +38,6 @@ public function instantiate(Fixture $fixture)
         $tablename = substr($fixture->getClass(), 6);
         $r = new TableRecord($tablename);
 
-        if (!isset($this->ids[$tablename])) {
```

Comment:
There are no option for this.
I just hope that in future we should not use id columns without autoincrement.
For old code where we have that fields - we just need to run old version of haigha (~1.0,<1.3.0)
Don't see another solution...